pkgs/by-name/ch/chatbox/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "chatbox";
+  version = "1.9.3";
+
+  src = fetchurl {
+    url = "https://download.chatboxai.app/releases/Chatbox-${version}-x86_64.AppImage";
+    hash = "sha256-7huaZC+vOO8nM0Mc48hMT6Jp0OCgC96yPzmNkBEir1w=";
+  };
+
+  appimageContents = appimageTools.extract { inherit pname version src; };
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    install -m 444 -D ${appimageContents}/xyz.chatboxapp.app.desktop $out/share/applications/${pname}.desktop
+    install -m 444 -D ${appimageContents}/usr/share/icons/hicolor/512x512/apps/xyz.chatboxapp.app.png $out/share/icons/hicolor/512x512/apps/${pname}.png
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace-fail 'Exec=AppRun' 'Exec=${pname}' \
+      --replace-fail 'Icon=xyz.chatboxapp.app' 'Icon=${pname}'
+  '';
+
+  meta = {
+    description = "Chatbox AI is an AI client application and smart assistant.";
```

```suggestion
    description = "AI client application and smart assistant";
```
To follow the style guide for meta.description